src/pages/CategoriesPage/UpdateCategory/UpdateCategoryComponent.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  Grid,
+  Button,
+} from '@mui/material';
+import TextField from '@mui/material/TextField';
+import { Formik } from 'formik';
+import { CreateUpdateCategoryModel } from '../../../models/categories/CreateCategoryModel';
+import { validationSchema } from '../../../validationSchemas/validationSchema';
+import { styles } from '../styles';
+
+interface Props {
+  updateModalOpen: boolean;
+  onToggleUpdateStatusClick: () => void;
+  onSubmit: (values: CreateUpdateCategoryModel) => void;
+  name: string;
+  description: string;
+}
+
+const UpdateCategoryComponent = (props: Props) => {
+  return (
+    <Dialog open={props.updateModalOpen}>
+      <div style={styles.dialogTitleWrapper}>
+        <DialogTitle sx={styles.dialogTitle}>
```

Comment:
Dialog should contain the right structure https://mui.com/material-ui/react-dialog/#ResponsiveDialog.tsx

Reply:
Dialog structure has been changed